### PR TITLE
Homework 0

### PR DESCRIPTION
**Pull request by BelierJavier**
First homework for nanos.

### CODE REVIEW

**Comment by BelierJavier** [comment]
@rickyc I have finished!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,6 @@
 # NHA-Repo
+<<<<<<< HEAD
 I am just a Read.meeee
```
Comment (rickyc) [inline README.md]: Great! We're almost done. So do you notice the `<<<<<<`, `======` and `>>>>>>` text? This means there was a merge conflict. We need to fix our code. 

`<<<<<< HEAD` is referring to the master branch or the branch that you want to merge `homework-0` into. `=====` is the midpoint so everything above it to `<<<<< HEAD` was code on the `master` branch. Everything below the `=====` is code in the `homework-0` branch.

So I want you to go back fix this file, then commit it and let me know when it's done.

**Comment by BelierJavier** [comment]
Woah you saw the commit I just did?

**Comment by BelierJavier** [comment]
I can't find the button

**Comment by rickyc** [comment]
😹. That's the power of a PR (pull request). Once you mention someone, they
get notifications of any changes that occur on the thread. So I can see
when you push work onto it. This lets me know if your work is done and I
can check it. You'll also receive notifications of any comments I make on
it :)

**Pull request merged**